linux_os/guide/system/auditing/policy_rules/audit_basic_configuration/ignition/shared.yml

```diff
@@ -8,7 +8,7 @@ spec:
     storage:
       files:
       - contents:
-          source: data:,%23%23%20First%20rule%20-%20delete%20all%0A-D%0A%0A%23%23%20Increase%20the%20buffers%20to%20survive%20stress%20events.%0A%23%23%20Make%20this%20bigger%20for%20busy%20systems%0A-b%208192%0A%0A%23%23%20This%20determine%20how%20long%20to%20wait%20in%20burst%20of%20events%0A--backlog_wait_time%2060000%0A%0A%23%23%20Set%20failure%20mode%20to%20syslog%0A-f%201%0A
```

Comment:
I'm afraid a mass-removal won't work. Each rule really has to be tested separately. E.g. here the audit_basic_configuration's check does expect a single newline. Before, there were two and my recent commit removed one of them.

A bigger issue is that some content is just incorrect, e.g. see PR#5651 where the urldecoded content would just be different from what the checks expect.